### DISCUSSION TITLE
采集容器日志文件时，如何添加元数据等信息？

### DISCUSSION THREAD

**Discussion by yangzw1024** [alibaba/ilogtail#1761] [Help] [answered]
在 k8s 集群中，我使用 DaemonSet 方式部署 ilogtail, k8s 中的 pod 会产生标准输出和记录日志到 hostpath 方式挂载日志盘中，如 /data/logs/{namespace}/{pod_name}/xxx.log, 我知道在采集标准输出时 ilogtail 可以通过配置自动添加上相关元数据，如 namespace, label 等，但是在采集 hostpath 路径下的日志时，如何添加相关元数据呢？ 比如 /data/logs/{namespace}/{pod_name}/xxx.log 路径中的   namespace 和 pod_name，以及 pod 的 label 等其它信息

**Comment by PalanQu** [comment, marked as answer]
可以通过ExternalK8sLabelTag来增加
